week-2/debug/0.js

```diff
@@ -5,3 +5,14 @@ function multiply(a, b) {
 }
 
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
+
+
+// The output will be 320 in the first console. This is because the function multiply has set an instruction inside the calibraces to multiply any two numbers that are given in the parameter of the function multiply.
+// However in the second console.log the result will be undefined because it is a local variable so we need to give it a return value for it to work.
```

Comment:
It isn't because it is a local variable. It is more because of what `console.log` does?
I recommend checking out the **Logging and returning** section from week 1 prep to help reinforce this concept: https://curriculum.codeyourfuture.io/js1/sprints/1/prep/

Reply:
Thank you. I have looked into it and updated my answer